jetbrains-core/tst/software/aws/toolkits/jetbrains/core/credentials/SettingsSelectorComboBoxActionTest.kt

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.credentials
+
+import com.intellij.testFramework.ProjectRule
+import com.intellij.testFramework.TestActionEvent
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Rule
+import org.junit.Test
+import software.aws.toolkits.core.credentials.aToolkitCredentialsIdentifier
+import software.aws.toolkits.core.region.anAwsRegion
+
+class SettingsSelectorComboBoxActionTest {
+    @Rule
+    @JvmField
+    val projectRule = ProjectRule()
+
+    private val DUMMY_REGION = anAwsRegion()
```

Comment:
The fuzzy-ing is nice

Reply:
yeah i think this is a pattern we should use more.